warehouse/ingest-core/src/test/java/datawave/ingest/mapreduce/EventMapperTest.java

```diff
@@ -212,6 +213,39 @@ public void errorEventWithZeroTimestampNotDropped() throws IOException, Interrup
         assertEquals(4, written.size());
     }
 
+    @Test
+    public void testDiscardInterval() throws IOException, InterruptedException {
+        // event date < now minus discard interval
+        long tomorrow = -1L * UIDConstants.MILLISECONDS_PER_DAY;
+        conf.setLong(record.getDataType().typeName() + "." + EventMapper.DISCARD_INTERVAL, tomorrow);
+
+        eventMapper.setup(mapContext);
+        eventMapper.map(new LongWritable(1), record, mapContext);
+        eventMapper.cleanup(mapContext);
+
+        Multimap<BulkIngestKey,Value> written = TestContextWriter.getWritten();
+
+        // discard interval lower bound is tomorrow. data from today should be dropped
+        assertEquals(0, written.size());
+    }
+
+    @Test
+    public void testFutureDiscardInterval() throws IOException, InterruptedException {
+        // event date > now plus future discard interval
+        long yesterday = -1L * UIDConstants.MILLISECONDS_PER_DAY;
```

Comment:
This test is functionally equivalent to testDiscardInterval.  Did you intend to use `EventMapper.DISCARD_FUTURE_INTERVAL` instead of `EventMapper.DISCARD_INTERVAL`?

Reply:
Right, I will fix that

Reply:
Done, see https://github.com/NationalSecurityAgency/datawave/pull/2271